docs/index.rst

```diff
@@ -11,16 +11,43 @@ Vulcanexus Documentation
   :target: http://www.vulcanexus.com/
 
 
-*Vulcanexus* is a *ROS 2* (Robot Operating System) all-in-one tool set.
+*Vulcanexus* is a *ROS 2* (Robot Operating System) eProsima extended distribution.
 It allows users to build robotics applications combining the unique *Vulcanexus* elements with the *ROS 2* libraries, having *Fast DDS* as its fixed middleware implementation.
 
 These open source elements include numerous features and tools, providing *Vulcanexus* users customizable solutions while improving overall system performance.
 With *Vulcanexus*, users have fast access to constantly improving functionalities, such as the latest *Fast DDS* version along with its new features.
 
-*Vulcanexus* collection of downloadable packages are:
+Getting started
+^^^^^^^^^^^^^^^
+
+* :ref:`Installation Manual <linux_binary_installation>`
+
+  - Instructions to set up ROS 2 with Vulcanexus for the first time.
+
+* :ref:`Overview <vulcanexus_global_introduction>`
+
+  - *Vulcanexus* concepts including *ROS 2* Documentation.
+
+* :ref:`Tutorials <tutorials_core_ros2_tutorials>`
+
+  - Collection of step-by-step instructions that help users to get started.
+
+* :ref:`Support <platforms>`
+
+  - Vulcanexus officially supported platforms and releases.
+
+* :ref:`Appendixes <glossary>`
+
+  - Glossary of terms used throughout this documentation.
+
+
+Vulcanexus project
+^^^^^^^^^^^^^^^^^^
+
+*Vulcanexus* is composed of a collection of downloadable packages:
 
 1. **VULCANEXUS-CORE**: a set of software libraries that enables users to build the most comprehensive and straightforward robotics application.
-   It consists of eProsima *Fast DDS* and *ROS 2*.
+   It consists of eProsima *Fast DDS* as middleware in *ROS 2*.
 2. **VULCANEXUS-TOOLS**: a set of features and applications which allows users to test, improve and configure the performance of *Vulcanexus* in their systems.
```

Comment:
I'd change the packages to lowercase format. This applies to all packages in this list.

```suggestion
2. **Vulcanexus Tools**: a set of features and applications which allows users to test, improve and configure the performance of *Vulcanexus* in their systems.
```

Reply:
Changed all of them.